instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/metrics/internal/DurationHistogramFactory.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.metrics.internal;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.unmodifiableList;
+
+import io.opentelemetry.api.metrics.DoubleHistogramBuilder;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.extension.incubator.metrics.ExtendedDoubleHistogramBuilder;
+import io.opentelemetry.instrumentation.api.internal.SemconvStability;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public final class DurationHistogramFactory {
```

Comment:
You'll need to add the following comment to public internal classes to fix the build:
```suggestion
/**
 * This class is internal and is hence not for public use. Its APIs are unstable and can change at
 * any time.
 */
public final class DurationHistogramFactory {
```